vortex-array/src/array/bool/accessors.rs

```diff
@@ -22,13 +22,7 @@ impl ArrayAccessor<bool> for BoolArray {
             Validity::Array(valid) => {
                 let valids = valid.into_bool()?.boolean_buffer();
                 let mut iter = valids.iter().zip(bools.iter()).map(|(is_valid, value)| {
-                    is_valid.then_some({
-                        if value {
-                            &TRUE
-                        } else {
-                            &FALSE
-                        }
-                    })
+                    is_valid.then_some(if value { &TRUE } else { &FALSE })
```

Comment:
this is just style but maybe this should also be `then`. Though don't think it matters